spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -201,14 +197,16 @@ function onChanOpenAck(
   // port has already been validated
   // assert that version is "ics27-1"
   abortTransactionUnless(version === "ics27-1")
+  // state change to keep track of successfully registered interchain account
+  setActiveChannel(SourcePortId)
 }
 ```
 
 ```typescript
 function onChanOpenConfirm(
```

Comment:
You also need `Close` callbacks since that needs to "unset" active channel. Is the plan that we create a new channel against the same portID to "recover" the account?

Reply:
yes exactly